float32 ratio
stroll_bearnav/FeatureArray map 
stroll_bearnav/FeatureArray view 
int32[] histogram
int32[] mapMatchIndex			#contains indices of viewFeatures corresponding to a given mapFeature
int32[] mapMatchEval			#contains information of mapFeatures - correct 1, incorrect -1, unmatched 0 
